### ISSUE TITLE
SHFileOperationW dialogs appear not focused on ST3

### ISSUE THREAD

**Issue by LeusMaximus** [aziz/SublimeFileBrowser#102]
When I want delete file with `shift+D` the dialog box is appear, but file not deleted.

**Comment by aziz** [comment]
I can't recreate this. It's working fine for me. 
Please provide more information like: 
- Sublime version
- FileBrowser version
- OS and OS version
- and the ST console output if you see any error there.

**Comment by LeusMaximus** [comment]
I was a little wrong. It working. I discovered this behavior:
- when I click `yes` in dialog box, opened another dialog-box (system dialog-box), but it isn't visible, because it appears behind sublime.
Windows7
Sublime 3083

**Comment by aziz** [comment]
@vovkkk can you look into this when you've got some time? I don't have Windows VM at the moment.

**Comment by vovkkk** [comment]
This is known, afaiu, it happen because plugins run in separate process which is in background so window created by this process is not focused, this explanation seems esp. correct because in ST2 plugins are in the same process as main window and created window become focused.
I do not know how forcefully focus dialog window, i.e. there are ofc winapi for this but I'm not sure how to use it in our context.

If anyone know how to fix focus, please, let us know or feel free to send pull request.

Speaking of system confirmation for send to recycle bin, specifically, I recommend to disable it, because all Windows newer than 7 do not ask by default.



-----Исходное сообщение-----
От: "Maksym" <notifications@github.com>
Отправлено: ‎28.‎10.‎2015 20:24
Кому: "aziz/SublimeFileBrowser" <SublimeFileBrowser@noreply.github.com>
Тема: Re: [SublimeFileBrowser] Delete File don't working (#102)

I was a little wrong. It working. I discovered this behavior:
when I click yes in dialog box, opened another dialog-box (system dialog-box), but it isn't visible, because it appears behind sublime. Windows7 Sublime 3083

**Comment by aziz** [comment]
> Speaking of system confirmation for send to recycle bin, specifically, I recommend to disable it, 
> because all Windows newer than 7 do not ask by default.

Why do we have it on by default then. I'd like this default behavior (which is done by ST natively is sidebar as well):

1. don't ask for confirmation by default 
2. move to trash (recycle bin) by default

**Comment by vovkkk** [comment]
> Why do we have it on by default then

Initially we hadn’t, but keybinding was <kbd>s</kbd> and you justly said that it is too easy to accidentally send to trash, so we agreed to enable confirmation by default; later you decide to turn all action keybindings to uppercase so it became <kbd>shift+d</kbd> and confirmation remained enabled.

I have no any strong opinion on that; one hand, disable confirmation to follow ST and Windows defaults; other hand, SFB with three view allow to mark dozens files so confirmation might be useful to see that you are about to send to trash more files than you expected.
Either way makes sense :confused:

**Comment by vovkkk** [comment]
It seems to fix system confirmation focus on 8.1, but does not affect other dialogues (progress, errors) 

```diff
--- dired_file_operations.py Sun Sep 13 15:26:16 2015
+++ dired_file_operations.py Thu Feb 18 12:56:36 2016
@@ -452,7 +452,11 @@
         SHFileOperationW.argtypes = [ctypes.POINTER(SHFILEOPSTRUCTW)]
         pFrom = u'\x00'.join(sources) + u'\x00'
         pTo   = (u'%s\x00' % destination) if destination else None
-        args  = SHFILEOPSTRUCTW(wFunc  = ctypes.wintypes.UINT(mode),
+        wf = ctypes.WINFUNCTYPE(ctypes.wintypes.HWND)
+        get_hwnd = wf(ctypes.windll.user32.GetForegroundWindow)
+        args  = SHFILEOPSTRUCTW(
+                                hwnd   = get_hwnd(),
+                                wFunc  = ctypes.wintypes.UINT(mode),
                                 pFrom  = ctypes.wintypes.LPCWSTR(pFrom),
                                 pTo    = ctypes.wintypes.LPCWSTR(pTo),
                                 fFlags = fFlags,
```

@LeusMaximus could you confirm?
